packages/picker/src/Picker.ts

```diff
@@ -417,6 +425,13 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
         `;
     }
 
+    public renderDescriptionSlot(): TemplateResult {
```

Comment:
Please make this a getter to match with other sub-templates in this file. It should also be `private` (if only used in this file) or `protected` (if leveraged in other class extensions).

Reply:
Done!